src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -135,6 +166,33 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
+    private async _mountCardVerificationfields(formOptions: HostedFormOptions) {
+        if (!formOptions) {
+            throw new InvalidArgumentError();
+        }
+
+        const { config } = this._store.getState();
+        const { paymentSettings: { bigpayBaseUrl: host = '' } = {} } = config.getStoreConfig() || {};
```

Comment:
should you be throwing an error if `host` is an empty string?

Reply:
When is empty it already show this error
![Screen Shot 2021-05-26 at 13 30 42](https://user-images.githubusercontent.com/69221626/119712879-b20a8280-be26-11eb-9b64-8c21d9bffb15.png)

Reply:
@icatalina 
if we remove the "dummy" value, its showing an alert about not providing a default value

![Screen Shot 2021-06-02 at 8 49 24](https://user-images.githubusercontent.com/69221626/120492303-926fde80-c37f-11eb-8a6a-124951fe7590.png)

Reply:
Hey @JoseAlvarezHdz!

I was thinking on something more like this:

```ts
const bigpayBaseUrl = config.getStoreConfig()?.paymentSettings.bigpayBaseUrl;

if (!bigpayBaseUrl) {
    throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
}

const form = this._hostedFormFactory.create(bigpayBaseUrl, formOptions);
```